spec/models/user_spec.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'factory_bot'
+require 'rails_helper'
```

Comment:
don't need in all the spec. put it in a common place

Reply:
@kth-wtag apu, done.